docs/api/safe-storage.md

```diff
@@ -0,0 +1,34 @@
+# safeStorage
+
+> Allows access to simple encryption and decryption of strings.
```

Comment:
I think we should write a bit more here about when this is the appropriate module to use. For instance, strings encrypted with this module probably aren't appropriate for sending over the network.

Perhaps:
```suggestion
> Allows access to simple encryption and decryption of strings for storage on the local machine.
```
?

Reply:
Will do, thank you ^^